src/main/java/net/minecraftforge/client/model/MultiModel.java

```diff
@@ -0,0 +1,135 @@
+package net.minecraftforge.client.model;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import javax.vecmath.Matrix4f;
+
+import net.minecraft.client.renderer.block.model.BakedQuad;
+import net.minecraft.client.renderer.block.model.ItemCameraTransforms;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.client.renderer.vertex.VertexFormat;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.model.BlockStateLoader.Version1;
+
+import com.google.common.base.Function;
+
+public class MultiModel implements IModel
+{
+    public static class Baked implements IFlexibleBakedModel
+    {
+        protected IFlexibleBakedModel base;
+        protected IFlexibleBakedModel[] parts;
+        
+        public Baked(IFlexibleBakedModel base, IFlexibleBakedModel[] parts)
+        {
+            this.base = base;
+            this.parts = parts;
+        }
+        
+        @Override
+        public boolean isAmbientOcclusion()
+        {
+            return base.isAmbientOcclusion();
+        }
+
+        @Override
+        public boolean isGui3d()
+        {
+            return base.isGui3d();
+        }
+
+        @Override
+        public boolean isBuiltInRenderer()
+        {
+            return base.isBuiltInRenderer();
+        }
+
+        @Override
+        public TextureAtlasSprite getTexture()
+        {
+            return base.getTexture();
+        }
+
+        @Override
+        public ItemCameraTransforms getItemCameraTransforms()
+        {
+            return base.getItemCameraTransforms();
```

Comment:
transforming the subparts individually is something that may come up in the future. Or not :P